src/V3AstNodeOther.h

```diff
@@ -699,6 +699,16 @@ class AstCFunc final : public AstNode {
                && finalsp() == nullptr;
     }
 };
+class AstCLocalScope final : public AstNode {
```

Comment:
Given where this is used shouldn't it be an AstNodeStmt?

Reply:
`AstVar` is not `AstNodeStmt` and it can be put in there. In fact putting `AstVar` inside this block is exactly what we want to use it for.

Reply:
I think I'll leave it for now as just `AstNode`. The reasoning behind it was that I want it to be treated in the most generic manner by all visitors that do not take any interest in it, as if it wasn't even there. If at some point there happens to be a use case for deriving from `AstNodeStmt` or `AstNodeBlock`, this could be changed.